### PR TITLE
sql/postgres: unmarshal spec column types

### PR DESCRIPTION
**Pull request by yonidavidson**
- convert column type

### CODE REVIEW

File: sql/postgres/sqlspec.go [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+	Schemas []*sqlspec.Schema `spec:"schema"`
+}
+
+// UnmarshalSpec unmarshals an Atlas DDL document using an unmarshaler into v.
+func UnmarshalSpec(data []byte, unmarshaler schemaspec.Unmarshaler, v interface{}) error {
+	var d doc
+	if err := unmarshaler.UnmarshalSpec(data, &d); err != nil {
+		return err
+	}
+	if v, ok := v.(*schema.Schema); ok {
```
Comment (a8m) [inline sql/postgres/sqlspec.go:25]: Keep the happy path unindented (same comment as for SQLite).

File: sql/postgres/sqlspec_test.go [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+	}
+	tmpl := `
+schema "default" {
+}
+table "table" {
+	schema = "default"
+	%s
+}
+`
+	body := fmt.Sprintf(tmpl, string(buf))
```
Comment (a8m) [inline sql/postgres/sqlspec_test.go:156]: ditto

File: sql/postgres/sqlspec.go (hunk 1 above)
Comment (yonidavidson) [reply sql/postgres/sqlspec.go:25]: 👍

File: sql/postgres/sqlspec_test.go (hunk 2 above)
Comment (yonidavidson) [reply sql/postgres/sqlspec_test.go]: 👍

**Review by a8m — APPROVED** [review]

**Pull request merged**